### PR TITLE
📖 Added basic storybook stories for amp-list

### PR DESCRIPTION
**Pull request by krdwan**
Added two stories in storybook for amp-list

### CODE REVIEW

**Comment by krdwan** [comment]
@wassgha A storybook story for amp-sidebar for your review.  I was unable to add custom CSS, do you have any information on how to add CSS to storybook?  If so, I may update this PR.

File: extensions/amp-sidebar/0.1/storybook/Basic.amp.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// To do: how to add CSS
+//import '!style-loader!css-loader!./Basic-styles.css';
```
Comment (wassgha) [inline extensions/amp-sidebar/0.1/storybook/Basic.amp.js:18]: See https://github.com/ampproject/amphtml/pull/27800

File: extensions/amp-sidebar/0.1/storybook/Basic.amp.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  .addParameters({extensions: [{name: 'amp-sidebar', version: 0.1}]})
+  .add('Basic Sidebar', () => {
+    return (
+      <div>
+        <div id="main">
+          <h1>Main Body</h1>
+          <p>
+            Use the buttons below to open / toggle the amp-sidebar. The sidebar
+            will be displayed with various navigational elements.
+          </p>
```
Comment (wassgha) [inline extensions/amp-sidebar/0.1/storybook/Basic.amp.js:39]: Usually examples should just be the component itself without any description. If we like we can use the storybook docs addon which allows for this style of inline examples. See https://github.com/storybookjs/storybook/tree/master/addons/docs#docspage
/cc @dvoytenko

Comment (krdwan) [reply extensions/amp-sidebar/0.1/storybook/Basic.amp.js:39]: Thanks for the input @wassgha .  I am happy to remove the description or move the entire story into a storybook doc.

One thing to note is that it appears that some components do not work in isolation.  For amp-list here, at a minimum a button outside of the component is needed to toggle the sidebar into view.

Sounds like we should establish a precedent on how we want to handle potential descriptions and other elements outside of the component in a story.

File: extensions/amp-sidebar/0.1/storybook/Basic.amp.js (hunk 1 above)
Comment (krdwan) [reply extensions/amp-sidebar/0.1/storybook/Basic.amp.js:18]: Thanks, I'll add a separate PR to include more use cases with CSS

File: extensions/amp-sidebar/0.1/storybook/Basic.amp.js (hunk 2 above)
Comment (krdwan) [reply extensions/amp-sidebar/0.1/storybook/Basic.amp.js]: Awesome!  Descriptions moved to comments as we discussed in latest commit.

File: extensions/amp-sidebar/0.1/storybook/Basic.amp.js (hunk 1 above)
Comment (dvoytenko) [reply extensions/amp-sidebar/0.1/storybook/Basic.amp.js:18]: The CSS pull request is approved.

**Pull request merged**